src/components/Admin/Admin.js

```diff
@@ -0,0 +1,108 @@
+import {React, Component} from 'react';
+import {Table, Tag, Space} from 'antd';
+
+/*Placeholder data*/
+const columns = [
+    {
+      title: 'First Name',
+      dataIndex: 'fname',
+      filters: [
+        {
+          text: 'Joe',
+          value: 'Joe'
+        },
+        {
+          text: 'Jim',
+          value: 'Jim'
+        }
+      ],
+      onFilter: (value, record) => record.fname.indexOf(value) === 0,
+      key: 'fname',
+      render: text => <a>{text}</a>,
+    },
+    {
+      title: 'Last Name',
+      dataIndex: 'lname',
+      key: 'lname',
+      render: text => <a>{text}</a>
+    },
+    {
+      title: 'Age',
+      dataIndex: 'age',
+      key: 'age',
+    },
+    {
+      title: 'Address',
+      dataIndex: 'address',
+      key: 'address',
+    },
+    {
+      title: 'Tags',
+      key: 'tags',
+      dataIndex: 'tags',
+      render: tags => (
+        <>
+          {tags.map(tag => {
+            let color = tag.length > 5 ? 'geekblue' : 'green';
+            if (tag === 'mentee') {
+              color = 'volcano';
+            }
+            return (
+              <Tag color={color} key={tag}>
+                {tag.toUpperCase()}
+              </Tag>
+            );
+          })}
+        </>
+      ),
+    },
+    {
+      title: 'Action',
```

Comment:
Looking over the wireframe, I think the only action that should be visible from the landing page is 'edit'. Deactivate/activate should be nested inside the accordion, beneath the form, and on a toggle.

Reply:
Ok, I will go ahead and remove the deactivate link and add the toggle inside the form. I will let you know when that is completed.